sdk/web/src/index.ts

```diff
@@ -0,0 +1,38 @@
+import { Picovoice } from './picovoice';
+import { PicovoiceWorker } from './picovoice_worker';
+
+import {
+  PicovoiceOptions,
+  PicovoiceWorkerDetectionResponse,
+  PicovoiceWorkerFailureResponse,
+  PicovoiceWorkerInferenceResponse,
+  PicovoiceWorkerInitRequest,
+  PicovoiceWorkerInitResponse,
+  PicovoiceWorkerProcessRequest,
+  PicovoiceWorkerProcessResponse,
+  PicovoiceWorkerResetRequest,
+  PicovoiceWorkerResetResponse,
+  PicovoiceWorkerReleaseRequest,
+  PicovoiceWorkerReleaseResponse,
+  PicovoiceWorkerRequest,
+  PicovoiceWorkerResponse,
+} from './types';
+
+export {
```

Comment:
does it make sense to also export some required types from the ppn and rhn packages? later in UI frameworks, it makes our life easier i guess